wrapanapi/systems/virtualcenter.py

```diff
@@ -228,6 +258,17 @@ def _pick_datastore(self, allowed_datastores):
             raise Exception("No possible datastores!")
         return possible_datastores[0]
 
+    def _pick_datastore_cluster(self):
+        """Pick a datastore cluster based on free space."""
+        possible_datastores = [dsc for dsc in self.system.get_obj_list(vim.StoragePod)
```

Comment:
Does `self.system.get_obj_list(vim.StoragePod)` return a list of datastores, or a list of datastore clusters? If the latter, `possible_datastores` should be called `possible_ds_clusters` or something similar

Reply:
It does return of list of datastore clusters, I can update the name.